bin/resources/GameIndex.yaml

```diff
@@ -7880,15 +7891,14 @@ SLES-50267:
     2547FF67:
       content: |-
         author=Nachbrenner
-        // Skip PSX2_Mpeg_PlayFile__FPc.
-        patch=0,EE,00107a00,word,03e00008
-        patch=0,EE,00107a04,word,24020001
         // Skip SuperSync__Fi.
         patch=0,EE,00100eb0,word,03e00008
         patch=0,EE,00100eb4,word,24020001
```

Comment:
It would be nice to check that game on the current IPU PR to see if we can get rid of the whole patch.

Reply:
What kind of bug is it?

Reply:
The game finishes a bunch of DMA's, then sets the PCR register to check VIF1, GIF and SIF1 to check for them being finished, then loops doing BC0F, which means it waits until all the DMA's have finished. All of them but GIF show as finished, because the game earlier cleared the fact the GIF had finished.  So it could be GIF is too fast? I dunno...